workshop/porter-tf-aci/azure/bundle/terraform/mysql.tf

```diff
@@ -1,42 +1,37 @@
 resource "random_string" "password" {
   length = 16
   special = true
-  override_special = "/@\" "
+  override_special = "/@£$"
```

Comment:
Why did this change?

Reply:
😂 the `"` was messing with my VSCode formatting... all lines below it weren't write... 'twas thinking it was all in one big quote.  Obvi more of an extension issue buuut I figured the actual override chars didn't really matter (pulled the new chars from the azurerm docs/examples).

Reply:
🖖 Carry on!